app/src/ui/repositories-list/repository-list-item.tsx

```diff
@@ -52,7 +59,34 @@ export class RepositoryListItem extends React.Component<
     const path = repository.path
     const gitHubRepo =
       repository instanceof Repository ? repository.gitHubRepository : null
-    const tooltip = gitHubRepo
+    const hasChanges = this.props.changedFilesCount > 0
+    const renderAheadBehindIndicator = () => {
+      if (!(repository instanceof Repository) || !this.props.aheadBehind) {
+        return null
+      }
+      const { ahead, behind } = this.props.aheadBehind
+      if (ahead === 0 && behind === 0) {
+        return null
+      }
+      const commitGrammar = (commitNum: number) =>
+        `${commitNum} commit${commitNum > 1 ? 's' : ''}` // english is hard
+      const aheadBehindTooltip =
+        'The currently checked out branch is' +
```

Comment:
Looks like we have to handle the case that `behind` is 0, as it stands, this would output _The currently checkout branch is its tracked branch._

Reply:
If `ahead` and `behind` are both 0, the `if` block right about this would have handled it and returned already.